src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -4,20 +4,20 @@ servers:
   - url: '/'
 
 tags:
-  - name: General Information
   - name: Announcements
-  - name: Attributes
   - name: Attribute Namespaces
+  - name: Attributes
   - name: Build
   - name: Configuration
   - name: Distributions
+  - name: General Information
   - name: Groups
   - name: Issue Trackers
   - name: Notifications
   - name: Person
   - name: Published Binaries
-  - name: Requests
-  - name: Sources
+  - name: Request
```

Comment:
Why is it changed to `Request`? Is it a typo? :thinking:

Reply:
It is, sorry :facepalm: I resolved a conflict and this slipped in